perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py

```diff
@@ -204,20 +204,20 @@ def GetConfig(user_config):
           FLAGS.gce_ssd_interface = FLAGS.server_gce_ssd_interface
         config['vm_groups']['clients']['vm_spec']['GCP']['num_local_ssds'] = 0
 
-  if FLAGS.aerospike_server_machine_type:
+  if AEROSPIKE_SERVER_MACHINE_TYPE:
     vm_spec = config['vm_groups']['workers']['vm_spec']
     for cloud in vm_spec:
-      vm_spec[cloud]['machine_type'] = FLAGS.aerospike_server_machine_type
-  if FLAGS.aerospike_client_machine_type:
+      vm_spec[cloud]['machine_type'] = AEROSPIKE_SERVER_MACHINE_TYPE
+  if AEROSPIKE_CLIENT_MACHINE_TYPE:
     vm_spec = config['vm_groups']['clients']['vm_spec']
     for cloud in vm_spec:
-      vm_spec[cloud]['machine_type'] = FLAGS.aerospike_client_machine_type
+      vm_spec[cloud]['machine_type'] = AEROSPIKE_CLIENT_MACHINE_TYPE
 
   if FLAGS['aerospike_vms'].present:
     config['vm_groups']['workers']['vm_count'] = FLAGS.aerospike_vms
 
-  if FLAGS['aerospike_client_vms'].present:
-    config['vm_groups']['clients']['vm_count'] = FLAGS.aerospike_client_vms
+  if AEROSPIKE_CLIENT_VMS.present:
```

Comment:
I don't think .present works with flag holders so maybe leave it as it was.

Reply:
Either leave it as it was or do:
if AEROSPIKE_CLIENT_VMS.value:
...